core/src/components/textarea/textarea.ionic.scss

```diff
@@ -0,0 +1,80 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@import "./textarea.common";
+@import "./textarea.ionic.vars";
```

Comment:
This import can be removed

Reply:
https://github.com/ionic-team/ionic-framework/pull/29776/commits/ea98cf34619813eebbd090272c124cdb515ba27b